src/Arcus.Scripting.IntegrationAccount/Scripts/Set-AzIntegrationAccountSchemas.ps1

```diff
@@ -0,0 +1,81 @@
+Param(
+    [Parameter(Mandatory = $true)][string] $ResourceGroupName = $(throw "Resource group name is required"),
+    [Parameter(Mandatory = $true)][string] $Name = $(throw "Name of the Integration Account is required"),
+    [parameter(Mandatory = $false)][string] $SchemaFilePath = $(if ($SchemasFolder -eq '') { throw "Either the file path of a specific schema or the file path of a folder containing multiple schemas is required, e.g.: -SchemaFilePath 'C:\Schemas\Schema.xsd' or -SchemasFolder 'C:\Schemas'" }),
+    [parameter(Mandatory = $false)][string] $SchemasFolder = $(if ($SchemaFilePath -eq '') { throw "Either the file path of a specific schema or the file path of a folder containing multiple schemas is required, e.g.: -SchemaFilePath 'C:\Schemas\Schema.xsd' or -SchemasFolder 'C:\Schemas'" }),
+    [Parameter(Mandatory = $false)][string] $ArtifactsPrefix = '',
+    [Parameter(Mandatory = $false)][switch] $RemoveFileExtensions = $false
+)
+
+if ($SchemaFilePath -ne '' -and $SchemasFolder -ne '') {
+    throw "Either the file path of a specific schema or the file path of a folder containing multiple schemas is required, e.g.: -SchemaFilePath 'C:\Schemas\Schema.xsd' or -SchemasFolder 'C:\Schemas'"
+}
+
+function UploadSchema {
+    param
+    (
+        [System.IO.FileInfo][parameter(Mandatory = $true)]$Schema
+    )
+
+    $schemaName = $Schema.Name
+    if ($RemoveFileExtensions) {
+        $schemaName = $Schema.BaseName
+    }
+    if ($ArtifactsPrefix -ne '') {
+        $schemaName = $ArtifactsPrefix + $schemaName
+    }
+    Write-Host "Uploading schema '$schemaName' into the Integration Account '$Name'"
+
+    ## Check if the schema already exists
+    $existingSchema = $null
+    try {
+        Write-Verbose "Checking if the schema '$schemaName' already exists in the Integration Account."
+        $existingSchema = Get-AzIntegrationAccountSchema -ResourceGroupName $ResourceGroupName -Name $Name -SchemaName $schemaName -ErrorAction Stop
+    }
+    catch {
+        if ($_.Exception.Message.Contains('could not be found')) {
+            Write-Verbose "Schema '$schemaName' could not be found."
+        }
+        else {
+            throw $_.Exception
+        }
+    }
+        
+    try {
+        if ($existingSchema -eq $null) {
+            # Create the schema
+            Write-Verbose "Creating schema '$schemaName'"
```

Comment:
Maybe some extra tracking info as separately, without context, it could become abstract.
```suggestion
            Write-Verbose "Creating schema '$schemaName' for Azure Integration Account '$Name'"
```